calendar-helper/index.js

```diff
@@ -102,9 +102,12 @@ class CalendarHelper {
 	}
 
 	getPage(m, inputDate) {
-		// I use a math clamp to check if the input date is in range
-		let dateNumber = Math.min(Math.max(inputDate, this.options.minDate), this.options.maxDate);
-		let date = new Date(dateNumber);
+		let dateNumber = this.options.minDate !== null || this.options.maxDate !== null ? Math.min(Math.max(inputDate, this.options.minDate), this.options.maxDate) : null;
+		let date = inputDate 
+		
+		if(dateNumber) {
+			date = new Date(dateNumber);
+		}
```

Comment:
This can be a ternary operator, to make it a little bit less imperative.
```suggestion
		let date = dateNumber ? new Date(dateNumber) : inputDate;
```